lib/layout/data_grid.dart

```diff
@@ -81,27 +81,18 @@ class DataGrid extends StatefulWidget
       'dividerThickness': (val) =>
           controller.dividerThickness = Utils.optionalDouble(val),
       'border': (Map val) {
```

Comment:
everywhere else we use borderRadius/borderWidth/borderColor. Can we make this consistent with that? borderRadius is commonly used but not borderWidth/borderColor, so put it in side another map is not ideal. Also nested style does not support binding

Reply:
make sure you use Utils's classes. Don't parse common properties yourself.

Reply:
> everywhere else we use borderRadius/borderWidth/borderColor. Can we make this consistent with that? borderRadius is commonly used but not borderWidth/borderColor, so put it in side another map is not ideal. Also nested style does not support binding currently.

Okay sure, I'll update it